CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+Our vision for Backstage is for it to become the trusted standard toolbox (read: UX layer) for the open source infrastructure landscape. Think of it like Kubernetes for developer experience. We realize this is an ambitious goal. We can’t do it alone.
+
+Therefore we want to create strong community of contributors -- all working together to create the kind of delightful experience that our developers deserve.
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given. ❤️
+
+# Types of Contributions
+
+## Report bugs
+
+Report bugs at https://github.com/spotify/backstage/issues.
```

Comment:
Avoid urls in text, make it "Report bugs [as issues](...)" or so

Reply:
Can link straight to the bug report template even! https://github.com/spotify/backstage/issues/new?template=bug_template.md